src/SDK.ts

```diff
@@ -92,6 +96,9 @@ export default class SDK implements ISDK {
     const response = await axiosInstance.get(endpoint);
     const elapsedTimeInMilliseconds = timer.milliSecondsElapsed;
     const { data } = response;
+    if(data.LiveChatVersion && data.LiveChatVersion == LiveChatVersion.V2){
```

Comment:
Use `===` instead of `==`

nit: space after `if` and before `{`

Reply:
Done